lang/en_US.json

```diff
@@ -613,5 +612,6 @@
     "Similar masteries": "Similar masteries",
     "Beaten by players of": "Beaten by players of",
     "Mastered by players of": "Mastered by players of",
-    "Roll again": "Roll again"
+    "Roll again": "Roll again",
+    "<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>": "<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>"
```

Comment:
If I customize this, it doesn't seem to update the checklist page.

Reply:
You should see an update if you try something like:
```json
"<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>": "<1>{{achievementTitle}}</1> 123from <2>{{gameTitle}}</2>"
```

Notice the "123from" in the value. The achievement title and game title are invisible.

{{achievementTitle}} and {{gameTitle}} come from the `values` prop:
```tsx
<Trans
  i18nKey="<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>"
  components={{
    1: <span className="sr-only" />,
    2: <span className="sr-only" />,
  }}
  values={{ achievementTitle: achievement.title, gameTitle: achievement.game.title }}
/>
```

Reply:
Yes, I'm aware that I should just be able to change the right side.
```
    "<1>{{achievementTitle}}</1> from <2>{{gameTitle}}</2>": "<1>{{achievementTitle}}</1> xxx <2>{{gameTitle}}</2>"
```

But I'm not seeing it in the UI.
![image](https://github.com/user-attachments/assets/b3c5fcbc-38c2-4bff-9c2d-aa97598f39a6)

Ideally, I'd like to test swapping the order: `"<2>{{gameTitle}}</2> featuring <1>{{achievementTitle}}</1>"`

Reply:
In latest, the `<Trans />` component now manipulates the label placement and your example should work.

In the original implementation, it was still moving elements, but the elements were invisible:
![Screenshot 2025-01-16 at 7 41 29 PM](https://github.com/user-attachments/assets/ecc26cf1-4276-49fe-a2a8-0674336e83cc)

Reply:
Ah, figured it out. Need to run `npm run build` to get it to pick up the changes.